<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSH / RDP Connector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>341</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connect">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>9</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Command</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRDSCommand">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>560</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Default Value</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>411</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabRDP">
     <attribute name="title">
      <string>RDP</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonSaveRdp">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>450</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save RDP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>HostName</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRDSHosname">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>118</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Domain</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>118</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRDSDomain">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRDSPassword">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxRDSSavePassword">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>381</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Resolution:</string>
      </property>
      <widget class="QLineEdit" name="lineEditRDSGeometryWidth">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditRDSGeometryHight">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Set geometry:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSFullscreen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>fullscreen</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSDisalesWallpaper">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>disables wallpaper</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSEnableDesktopComposition">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>enable desktop composition</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Performance flags:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSEnableCompression">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>enable compression</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxRDSPerfFlags">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Broadband</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Modem</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Security:</string>
      </property>
      <widget class="QCheckBox" name="checkBoxRDSForceNTLM">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>341</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>force NTLM authentication protocol version (1 or 2)</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSNoRdp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>no-rdp</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSNoTls">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>no-tls</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSNoNla">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>no-nla</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRDSIgnoreCertificate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ignore-certificate</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Other Cmd Line Arg</string>
      </property>
      <widget class="QLineEdit" name="lineEditRDSOtherCmdLine">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>361</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonRDSHelp">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSSH">
     <attribute name="title">
      <string>SSH</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonSaveSSH">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>680</y>
        <width>152</width>
        <height>43</height>
       </rect>
      </property>
      <property name="text">
       <string>Save SSH</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Cmd Output</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuConfiguration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
